docs/ibc/proto-docs.md

```diff
@@ -785,6 +786,25 @@ Packet defines a type that carries data across different chains through IBC
 
```

Comment:
I think we need to modify something to get the ics29 proto files to be added

Reply:
ye just noticed this also\

Reply:
Need to increase max depth in proto script